README.md

```diff
@@ -11,9 +11,28 @@ If there's any issue you are facing in setting up this theme I'm there for you.
 
 
 
-![alt-tag](https://farm2.staticflickr.com/1674/25549273413_3872aa3b1e_o.png)
-
-
+# POSTING
+- Create a .markdown file inside `_posts` folder.
+- Name the file according to the format YY-MM-DD-[short name for your post].
+- `2016-03-30-i-love-design.markdown`
+- Write the _Front Matter_ and content in the file.
+
+# FORMAT
```

Comment:
@jcardoz minor change, presently the POSTING and FORMAT are of same size, so you can decrease the font size of FORMAT so that it can be differentiated from the topic (POSTING) and can be perceived as a subtopic.